lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -109,6 +109,8 @@ import qualified Cardano.Api.Typed as Cardano
 import qualified Cardano.Chain.Common as Byron
 import qualified Cardano.Crypto as CC
 import qualified Cardano.Crypto.Hash.Class as Crypto
+import Cardano.Crypto.Wallet
+    ( XPub )
```

Comment:
neat-picking: this import should not be there. We group imports as:

Prelude
Explicit imports
Qualified imports